nih-sparc/sparc-app
# Organ details page - Display related Datasets #4a9h1e

# Description

The purpose of this PR is to display the related datasets for an organ.

## Type of change


# How Has This Been Tested?

- Visit the Find Data page
- Click on the 'Organs' tab
- Click on the 'Heart' organ
- You should be taken to that organ's page
- Click on the 'Datasets' tab
- You should see the related datasets to that organ
- Click on one of the datasets' names
- You should be taken to that dataset's page

# Checklist:
